farmer.js

```diff
@@ -88,13 +90,27 @@ function checkMinPlaytime() {
 			g_Jar.setCookie(cookie, 'https://steamcommunity.com');
 		});
 		
-		request("https://steamcommunity.com/my/badges/", function(err, response, body) {
+		request("https://steamcommunity.com/my/badges/?p="+g_Page, function(err, response, body) {
 			if(err || response.statusCode != 200) {
 				log("Couldn't request badge page: " + (err || "HTTP error " + response.statusCode) + ". Retrying in 10 seconds...");
-				setTimeout(checkMinPlaytime, 10000);
+				setTimeout(checkMinPlaytime(g_Page), 10000);
 				return;
 			}
 			
+			g_Html = g_Html+body;
+			var $ = Cheerio.load(g_Html);
+			log("Page "+g_Page+" Loaded!");
+			
+			if ( $('.badge_row').length/250 == Math.round($('.badge_row').length/250) ){
+						g_Page++;
+						log("Checking page "+g_Page+" for drops");
+						checkMinPlaytime(g_Page);	
+						return;
+			}
+			
+			log(""+$('.badge_row').length+" apps with badges found");
+			log("Checking app playtime...");
```

Comment:
Moved log("Checking app playtime..."); to here so it will not be repeated each time a new page is checked.